.travis.yml

```diff
@@ -18,6 +18,7 @@ script:
   - mix coveralls.travis --max-cases 1
   - mix credo
   - mix dialyzer --halt-exit-status
+  - mix format --check-formatted
```

Comment:
Could we change https://github.com/lob/lob-elixir/blob/master/CONTRIBUTING.md to mention formatting as well?